Split out configuration settings by task/outcome (#5787)
mattermost/docs: source/configure/file-storage-configuration-settings.rst

  :target: https://mattermost.com/deploy
  :alt: Available for Mattermost Self-Hosted deployments.

:nosearch:

Configure file storage settings by going to **System Console > Environment > File Storage**, or by editing the ``config.json`` file as described in the following table. 

> Mattermost currently supports storing files on the local filesystem and Amazon S3 or S3-compatible containers. We have tested Mattermost with `MinIO <https://min.io/>`__ and `Digital Ocean Spaces <https://docs.digitalocean.com/products/spaces/>`__ products, but not all S3-compatible containers on the market. If you are looking to use other S3-compatible containers, we recommend completing your own testing.

Reviewer: ```suggestion
Mattermost currently supports storing files on the local filesystem and Amazon S3 or S3-compatible containers. We've tested Mattermost with `MinIO <https://min.io/>`__ and `Digital Ocean Spaces <https://docs.digitalocean.com/products/spaces/>`__ products, but not all S3-compatible containers on the market. If you're looking to use other S3-compatible containers, we recommend completing your own testing.
```
